### ISSUE TITLE
pystan compilation error

### ISSUE THREAD

**Issue by jbloom** [stan-dev/pystan#207]
I was using `pystan` version 2.5 without incident. I then used `pip` to upgrade to `pystan` version 2.9.0.0 and `Python` version 2.7.6. Now I cannot get even a simple model to compile. For instance, even when I use the simple *schools_code* example at https://pypi.python.org/pypi/pystan I get the error message below. I am using `gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04)`

Here is the error message:

     INFO:pystan:COMPILING THE C++ CODE FOR MODEL anon_model_7af71fe73cad6b7d51a4ce15383db4c6 NOW.
    Traceback (most recent call last):
    File "temp.py", line 31, in <module>
       iter=1000, chains=4)
    File "/home/jbloom/.local/lib/python2.7/site-packages/pystan/api.py", line 368, in stan
       verbose=verbose)
    File "/home/jbloom/.local/lib/python2.7/site-packages/pystan/model.py", line 312, in __init__
       build_extension.run()
    File "/usr/lib/python2.7/distutils/command/build_ext.py", line 337, in run
       self.build_extensions()
    File "/usr/lib/python2.7/distutils/command/build_ext.py", line 446, in build_extensions
       self.build_extension(ext)
    File "/usr/lib/python2.7/distutils/command/build_ext.py", line 496, in build_extension
       depends=ext.depends)
    File "/usr/lib/python2.7/distutils/ccompiler.py", line 574, in compile
       self._compile(obj, src, ext, cc_args, extra_postargs, pp_opts)
    File "/usr/lib/python2.7/distutils/unixccompiler.py", line 122, in _compile
       raise CompileError, msg
    distutils.errors.CompileError: command 'x86_64-linux-gnu-gcc' failed with exit status 1

**Comment by ariddell** [comment]
That sounds like a good gcc version. Can you try compiling something with ``debug=True``? Also, can you verify that you're using Cython>=0.22?

**Comment by jbloom** [comment]
Hi Allen,

I am using `cython` version 0.23.4.

I couldn't find the `debug` option, not sure if it is the same as `verbose`? Anyway, I tried re-running the example at https://pypi.python.org/pypi/pystan but added `verbose=True` to the `pystan.stan` command.

I have attached the standard error and standard output. It is not obvious to me from this message what is causing the problem, but you may be able to interpret it better...

Thanks,
Jesse
[error.txt](https://github.com/stan-dev/pystan/files/96029/error.txt)

**Comment by ariddell** [comment]
yes, verbose (not debug).

Did you install from source? It looks like there's some issue with the stan source.

**Comment by jbloom** [comment]
Hi Allen,

I installed using `pip`. However, I now realized that due to some issues with the paths to my `pip` installed packages, `pystan` was using an older version of `cython`. When I manually removed all versions of `pystan` and `cython` and then re-installed with `pip`, everything worked fine.

So it looks like the problem was on my end rather than anything to do with `pystan`. Thanks for the help, and sorry for wasting your time on this...

--Jesse
